packages/api/src/controllers/room.ts

```diff
@@ -0,0 +1,34 @@
+import { Router } from "express";
+import { RoomServiceClient } from "livekit-server-sdk";
+import { v4 as uuid } from "uuid";
+import { authorizer } from "../middleware";
+
+const app = Router();
+
+app.post("/", authorizer({}), async (req, res) => {
+  if (req.config.livekitHost == "") {
+    res.status(400);
+    return res.send("not enabled");
+  }
+
+  const id = uuid();
+  const svc = new RoomServiceClient(
```

Comment:
Is it ok to create one of these on every request? If there are definite gains in having a single instance lmk and we can work sth out, like putting it on the `req` object as well

Reply:
Yeah I wondered that too, when I asked livekit they said it should be ok:
> either way works! it's pretty inexpensive to create each time

Sound ok to you or should we press further?

Reply:
Sounds good for now!